5.37.0/upgradeLegacyRteToCmsFeatureFlag.ts

```diff
@@ -0,0 +1,38 @@
+import { Files } from "../utils";
+import { Project } from "ts-morph";
+import { Context } from "../types";
+import { isFeatureFlagExist } from "../utils/isFeatureFlagExist";
+import { addFeatureFlag } from "../utils/addFeatureFlag";
+
+interface Params {
+    files: Files;
+    project: Project;
+    context: Context;
+}
+
+export const upgradeLegacyRteToCmsFeatureFlag = async (params: Params) => {
+    const { project, files, context } = params;
+
+    context.log.info(
+        "Adding feature flag for legacy Rich Text editor support in the Headless CMS app."
+    );
+
+    const webinyProjectFile = files.byName("webiny.project.ts");
+    const source = project.getSourceFile(webinyProjectFile.path);
+
+    const FEATURE_FLAG_NAME = "allowCmsLegacyRteInput";
```

Comment:
Can we have this ~L12 like we did it [here](https://github.com/webiny/webiny-upgrades/pull/23/files#diff-a89b7bf3a420157538c4f65a9ee741a46b8f4e9cbf0acbd82a015023a87e5f3bR5)?